src/medCore/data/medAbstractDataWriter.h

```diff
@@ -0,0 +1,23 @@
+#ifndef MEDABSTRACTDATAWRITER
+#define MEDABSTRACTDATAWRITER
+
+#include <dtkCore/dtkAbstractDataWriter.h>
+#include <medCoreExport.h>
+
+
+class medAbstractData;
+/**
+ * Extending dtkAbstractDataWriter class to allow the export of a list of data
+ */
+class MEDCORE_EXPORT medAbstractDataWriter : public dtkAbstractDataWriter
+{
+    Q_OBJECT
+
+public:
+    using dtkAbstractDataWriter::setData;
+    virtual void setData(QList<medAbstractData*> data) {}
```

Comment:
I did another PR to correct that : https://github.com/Inria-Asclepios/medInria-public/pull/209